redisinsight/api/test/api/z-set/PATCH-instance-id-zSet.test.ts

```diff
@@ -22,9 +22,9 @@ const dataSchema = Joi.object({
   member: Joi.object().keys({
     name: Joi.string().required(),
     // todo: allow(true) - is incorrect but will be transformed to number by BE. Investigate/fix it
-    score: Joi.number().required().allow(true),
+    score: Joi.number().required().allow('inf', '-inf').label('.score'),
```

Comment:
it is better to create own type here the same way as for `redisString` so it will look like `Joi.zSetScore().required()` or something similar